aws-sagemaker-python/README.md

```diff
@@ -0,0 +1,82 @@
+# AWS SageMaker + Hugging Face LLM Deployment
```

Comment:
Let's change this to "Amazon SageMaker" as well (and yes, I know the template name is `sagemaker-aws-python`...don't point out the inconsistencies)

Reply:
resolved in https://github.com/pulumi/templates/pull/643/commits/c4d9a4d56a45c5b2d638254a92ea89542a457945